demisto_sdk/tests/test_files/swagger_python.txt

```diff
@@ -159,6 +159,28 @@ class Client(BaseClient):
         return response
 
 
+    def post_pet_upload_image_request(self, petId, additionalMetadata, file):
+        data = assign_params(additionalMetadata=additionalMetadata, file=file)
+
+        headers = self._headers
+        headers['Content-Type'] = 'multipart/form-data'
+
+        response = self._http_request('post', f'pet/{petId}/uploadImage', json_data=data, headers=headers)
+
+        return response
+
+
+    def post_pet_upload_image_uploadimage_request(self, petId, additionalMetadata, file):
```

Comment:
the name looks weird. is it supposed to be like that?

Reply:
yes, its the only way to handle this kind of duplications.